gravitee-apim-cypress/cypress/model/am_applications.ts

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
maybe we can create a dedicated folder for am model ?
something like `model\apim` folder and `model\am` folder.

WDYT ?

Reply:
Yes, makes sense and we could do so later on. However, as long as we don't have a final decision on how we deal with with these (cross-platform) policy tests I'd like to leave the structure as it is. We don't have a folder structure for the existing interfaces definitions either and the prefix am_ is probably sufficient for now. I hope that's ok for you.